src/test/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -90,7 +88,7 @@ public void testDisconnect() throws Exception {
     SmartcarResponse res = new SmartcarResponse(data);
 
     PowerMockito.doReturn(res.toString())
-      .when(this.subject, "call", "disconnect", "DELETE", null);
+      .when(this.subject, "call", "application", "DELETE", null);
```

Comment:
Why did this change?

Reply:
It was wrong, the correct endpoint is `DELETE /vehicles/:id/application/`where it was originally hitting `DELETE /vehicles/:id/disconnect`